.eslintrc.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
     'import/order': 'off',
     'no-await-in-loop': 'off',
     'no-useless-constructor': 'off',
-    'no-empty-function': 'warn',
+    'no-empty-function': 'off',
```

Comment:
this was requiring a lot of `eslint-disable-next-lines` and just seemed like we don't really need this on.